packages/account/src/Sections/Security/ConnectedApps/connected-apps-info-bullets.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { CONNECTED_APPS_INFO_BULLETS } from 'Constants/connected-apps-config';
+
+type TConnectedAppsInfoBulletsProps = {
+    class_name: string;
+    text_color?: string;
```

Comment:
```suggestion
    text_color?: React.CSSProperties['color'] | string
```

Reply:
`text_color` here is actually a part of predefined color variable name. in all the usage it will be a string.